man/fi_domain.3.md

```diff
@@ -506,6 +508,80 @@ the default attributes of the domain.
 This is the maximum number of IO vectors (scatter-gather elements)
 that a single memory registration operation may reference.
 
+## Capabilities (caps)
+
+Domain level capabilities.  Domain capabilities indicate the
+supported features of the domain, and the reported values are
+independent of those specified through the fi_info structure.
+See the CAPABILITIES section of fi_getinfo(3) for capability details.
+If the caps field is 0 on input to fi_getinfo(3), the caps value from the
+fi_info structure will be used to select an appropriate domain.
+
+Domain capabilities are relative to the endpoint types supported by
+the domain.  A domain capability indicates a basic set of functionality.
+If a capability bit is set, then the domain supports that
+capability for all endpoint types that are both supported by the domain
+and are listed for that capability as listed below.  Setting
+a capability bit does not guarantee support for a specific endpoint type.
```

Comment:
Need client use cases for how either would be usable. Suggestion is to define caps/mode bits that only apply to the domain, rather than try to map all endpoint capabilities to the domain.  E.g. SHARED_AV support can work fine as a domain capability.